lib/mayaUsd/ufe/UsdUndoAddNewPrimCommand.cpp

```diff
@@ -115,6 +120,11 @@ std::string UsdUndoAddNewPrimCommand::commandString() const
     return std::string("CreatePrim ") + _primToken.GetText() + " "
         + Ufe::PathString::string(_newUfePath);
 }
+
+Ufe::SceneItem::Ptr UsdUndoAddNewPrimCommand::sceneItem() const
+{
+    return Ufe::Hierarchy::createItem(newUfePath());

```

Comment:
This method is currently not used or tested. However, I'm not sure if it can easily be tested right now, since `UsdUndoAddNewPrimCommand` is tested though `Ufe::ContextOps::doOpCmd()` (see [here](https://github.com/Autodesk/maya-usd/blob/60d70d6e7ccb844d920e051fd98639d5ba9f95fd/test/lib/ufe/testContextOps.py#L314)), which returns a `UndoableCommand::Ptr`, not a `Ufe::SceneItemResultUndoableCommand()`.

Reply:
I made the material commands call `UsdUndoAddNewPrimCommand::sceneItem()` instead of manually calling `Ufe::Hierarchy::createItem()` on `UsdUndoAddNewPrimCommand::newUfePath()`. This makes the code a little more tidy and ensures that `UsdUndoAddNewPrimCommand::sceneItem()` is covered by existing tests. See 088f54ab81ccfb4ddc941d5c120888d34dee9fea.